testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -77,6 +82,25 @@
             // (no explicit configuration, none or multiple JDBC driver extensions)
             // Result of DevServices support https://github.com/quarkusio/quarkus/pull/14960
             Pattern.compile(".*Unable to determine a database type for default datasource.*"),
+    }),
+    WINDOWS_COMMON(new Pattern[] {
+            // When running on Windows, some warning SLF4J traces are randomly printed
+            Pattern.compile(".*SLF4J: .*")
+    }),
+    WINDOWS_DEV_MODE_ERRORS(new Pattern[]{
+            Pattern.compile(".*Re-run Maven using the -X switch to enable full debug logging.*"),
+            Pattern.compile(".*For more information about the errors and possible solutions, please read the following articles.*"),
+            Pattern.compile(".*Failed to run: Dev mode process did not complete successfully.*"),
+            Pattern.compile(".*http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException.*"),
+            Pattern.compile(".*To see the full stack trace of the errors, re-run Maven with the -e switch.*"),
+            Pattern.compile("\\[ERROR\\] *"),
+    }),
+    WINDOWS_NATIVE(new Pattern[]{
+            Pattern.compile(".*Uber JAR strategy is used for native image source JAR generation on Windows.*"),
+            Pattern.compile(".*Duplicate entry about.html entry.*"),
```

Comment:
Why is this Windows specific?

Reply:
Moved to common platform whitelist log lines.